### ISSUE TITLE
Issue inviting non workspace members to channels

### ISSUE THREAD

**Issue by yacjne** [The-Commit-Company/raven#1180]
Hey there,

I’ve noticed something unusual with Workspaces/Channels in Raven: 
- In a private workspace, I can currently invite users who aren’t members of that workspace to channels. These users technically become members of the channel, but won’t see it because they aren’t part of the workspace itself.

Possible solutions might be:

Automatically add invited users to the workspace when they join a private channel.
Restrict adding users who aren’t already part of the workspace.

Let me know what you think. Thanks!

Cheers, yacjne.

**Comment by nikkothari22** [comment]
@yacjne Yes, I think we have not filtered the list of members in the add channel popup as of now. Will do that.
